autosubmit/job/job_list.py

```diff
@@ -1487,11 +1481,11 @@ def _manage_job_dependencies(self, dic_jobs, job, date_list, member_list, chunk_
                 dependencies_of_that_section.append(key_aux_stripped)
             # Adds the dependencies to the job, and if not possible, adds the job to the
             # problematic_dependencies
+            ## Adds the dependencies to the job, and if not possible, adds the job to the problematic_dependencies
```

Comment:
Single `#` instead of `##`. PyCharm warns about it... Please check that your PyCharm is properly configured :+1: 

![image](https://github.com/user-attachments/assets/b274a21c-a75e-481f-b9dc-2a41c6987756)